docker-compose.yml

```diff
@@ -47,5 +47,34 @@ services:
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock
     command: --interval 300 --debug
+  keycloak-db:
+    image: "postgres:12.2-alpine"
+    restart: always
+    environment:
+      POSTGRES_DB: keycloak
+      POSTGRES_USER: keycloak
+      POSTGRES_PASSWORD: password
+    restart: on-failure
+    stdin_open: true
+    tty: true
+    volumes:
+      - "/docker-entrypoint-initdb.d/docker/data/keycloak-db:/var/lib/postgresql/data/"
+  keycloak:
+    image: quay.io/keycloak/keycloak:latest
+    volumes:
+      - "/docker-entrypoint-initdb.d/docker/data/keycloak:/opt/jboss/keycloak/imports"
```

Comment:
ditto. Moreover, `realm-export.json` are missed

e.g

```yml
keycloak:
    image: quay.io/keycloak/keycloak:latest
    volumes:
      - ./imports:/opt/jboss/keycloak/imports
    environment:
      KEYCLOAK_IMPORT: /opt/jboss/keycloak/imports/realm-export.json -Dkeycloak.profile.feature.upload_scripts=enabled       
```

Reply:
What ditto?
`realm-export.json` defined in
`KEYCLOAK_IMPORT: /opt/jboss/keycloak/imports/realm-export.json -Dkeycloak.profile.feature.upload_scripts=enabled`

Reply:
I suggest you to read docker manual, you seem to have no real understanding how volumes work and what `docker-entrypoint-initdb.d` means.

Reply:
ok, I'll do it, but it works now. May you write what the problem is? I really don't understand that I should `ditto`.

Reply:
You use default realm and default credentials... obviously, keycloak just uses default config.

> I really don't understand that I should ditto.

 `docker-entrypoint-initdb.d` is misused like in `keycloak-db` service